src/tests/functional/inference_engine/transformations/nop_elimination.cpp

```diff
@@ -817,4 +820,214 @@ TEST(nop_elimination, gather_3d_indices_constant_axis_1) {
+            std::ostringstream result;
+            result << type_params.op_type
+                   << "_input1=" << shape_params.shape1
+                   << "_input2=" << shape_params.shape2
+                   << "_swap_inputs=" << std::boolalpha << shape_params.swap_inputs
+                   << "_constant=" << type_params.constant_kind
+                   << "_type=" << element_type;
+            return result.str();
+        }
+};
+
+TEST_P(EliminateEltwiseTests, eliminate_eltwise) {
+    auto params = GetParam();
+    const auto& shape_params = std::get<0>(params);
+    const auto& type_params = std::get<1>(params);
+    const auto& type = std::get<2>(params);
+    const auto& shape1 = shape_params.shape1;
+    const auto& shape2 = shape_params.shape2;
+    bool swap_inputs = shape_params.swap_inputs;
+    bool can_fuse = shape_params.can_fuse && type_params.can_fuse;
+
+    auto parameter = make_shared<op::Parameter>(type, shape1);
+    std::shared_ptr<Node> constant;
+    switch (type_params.constant_kind) {
+        case ConstantKind::ZERO:
+            constant = op::Constant::create(type, shape2, {0});
+            break;
+        case ConstantKind::ONE:
+            constant = op::Constant::create(type, shape2, {1});
+            break;
+        case ConstantKind::RANDOM:
+            constant = builder::makeConstant(type, shape2, {}, true, 20 /* upTo */, 2 /* startFrom */);
+            break;
+    }
+
+    shared_ptr<Node> A = parameter;
+    shared_ptr<Node> B = constant;
+    if (swap_inputs) {
+        std::swap(A, B);
+        if (type_params.op_type == EltwiseTypes::SUBTRACT ||
+            type_params.op_type == EltwiseTypes::DIVIDE) {
+            can_fuse = false;
+        }
+    }
+
+    shared_ptr<Node> node;
+    switch (type_params.op_type) {
+        case EltwiseTypes::ADD:
+            node = make_shared<opset8::Add>(A, B);
+            break;
+        case EltwiseTypes::SUBTRACT:
+            node = make_shared<opset8::Subtract>(A, B);
+            break;
+        case EltwiseTypes::MULTIPLY:
+            node = make_shared<opset8::Multiply>(A, B);
+            break;
+        case EltwiseTypes::DIVIDE:
+            node = make_shared<opset8::Divide>(A, B);
+            break;
+        default:
+            ASSERT_FALSE(true) << "Invalid EltwiseType";
+    }
+    auto abs = make_shared<opset8::Abs>(node);
+    function = make_shared<Function>(abs, ParameterVector{parameter});
+
+    manager.register_pass<pass::NopElimination>();
+
+    if (can_fuse) {
+        auto abs = make_shared<opset8::Abs>(parameter);
+        function_ref = make_shared<Function>(abs, ParameterVector{parameter});
+    }
+
+    comparator.enable(FunctionsComparator::CmpValues::CONST_VALUES);
+    if (type == element::f32) {
+        enable_accuracy_check();
+    }
+}
+
+std::vector<ShapeParams> shape_params = {
+    // input 1, input 2, swap inputs, can fuse
+    { Shape{}, Shape{}, false, true },
+    { Shape{}, Shape{}, true, true },
```

Comment:
I have some doubts about `ov::is_scalar` method which is used inside `can_eliminate_eltwise_node` function, so could you please add `{ Shape{}, Shape{1}, false, false }` test case? it should be eliminated as second shape [1] broadast first input shape [] to [1].